README.md

```diff
@@ -265,6 +266,18 @@ android {
 }
 ```
 
+### Enable Detekt checks only on changed files
+The plugin provides the ability to check only changed files. You can enable this:
```

Comment:
The description should be more detailed.

```suggestion
Plugin `redmadrobot.detekt` adds task `detektDiff` to check the only files changed comparing to the base branch.
To enable this feature, you should specify base branch name:
```

Reply:
Applied you suggestion.